src/ripple/app/tx/impl/Change.cpp

```diff
@@ -221,4 +235,127 @@ Change::applyFee()
     return tesSUCCESS;
 }
 
+TER
+Change::applyUNLModify()
+{
+    if (view().seq() % 256 != 0)
```

Comment:
We might want a constant for the flag interval, or an "isFlagLedger` function. I'm not worried about the interval changing, but it makes the code clearer to see `isFlagLedger(view().seq())` rather than `% 256`

Reply:
Added `isFlagLedger` function. Please take a look.